cvrf2csaf/common/utils.py

```diff
@@ -22,6 +39,11 @@ def get_config_from_file() -> dict:
         path_to_conf = pkg_resources.resource_filename(req, 'cvrf2csaf/config/config.yaml')
         with open(path_to_conf, 'r') as f:
             config = yaml.safe_load(f)
+
+        for key in ['force']:
```

Comment:
Nit: YAGNI, the sequence?

Reply:
`--force-update-revision-history` will probably gonna need it, when added to the `config.yaml`.

Reply:
Integrated.